SU-HSDO/suhumsci
# feat: added new color pairings to the colorful theme

I added the following color pairings to the colorful theme options:
- Lake
- Canyon
- Cliff

# READY FOR REVIEW

## Summary
- Color pairings Lake, Canyon and Cliff for the Colorful theme have been added to the options in the color settings list. 
- I also updated the color variable for `hover border bottom` across the colorful theme, changing from `tertiary-reversed` to` tertiary-highlight-darken-10`. This is also a change reflected in the color pairing spreadsheet as well.

## Steps to Test
1. clone branch
2. npm test
3. npm run build
4. Swap color settings in (Appearance > Humsci Colorful > Settings > Color Pairing) swap between the different color settings and compare these themes to the Figma file and make sure they look accurate. You can also confirm the hex codes in the spreadsheet to what's on the site for comparison.

Figma Design:
https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme-Working?node-id=1066%3A2637

Color Pairing Spreadsheet:
https://docs.google.com/spreadsheets/d/1m2Dt4trd2qLZJhsuXidLBtTRovlm4tLwRyG3ic07ANM/edit?usp=sharing

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
